### DISCUSSION TITLE
软件奶妈 | /cn/2015/10/software-nanny/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#841] [Comments]
https://yihui.org/cn/2015/10/software-nanny/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我这篇文章的意思是欢迎提问，但不要一上来就问作者，而是先到社区问，例如StackOverflow或COS论坛。如果等了几天社区没有人回答你，你可以尝试联系作者，并附上你在社区提问的帖子链接。如果作者有精力回答，他可以到社区回答，这样也便于将来其他人有类似问题可以在公共场所找到答案，避免一个人累死八回还要一遍遍处理重复的邮件。把软件作者当私人顾问是非常低效的获取知识方式。

从我的角度，我希望能帮提问者节省时间（没必要独自折腾八个小时，该问还是要问，也许答案很简单呢），也希望提问者能帮我节省时间。

> ——原帖发布于 2015-11-09 20:51:07

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
好问题，我刚在这里解答了：<https://www.youtube.com/watch?v=U2-O4D15x3A> 代码在此：<https://gist.github.com/yihui/836f9c995d37776ea62c>

> ——原帖发布于 2016-01-17 03:29:30

[a comment by an automated account is not shown]

**Reply by think-crow** [reply]
哈哈哈哈哈哈哈

[a comment by an automated account is not shown]
